.vscode/launch.json

```diff
@@ -6,7 +6,7 @@
             "type": "node",
             "request": "attach",
             "port": 9229,
-            "preLaunchTask": "func: host start"
+            "preLaunchTask": "func: host start --cors *"
```

Comment:
Why do we need now `--cors`?

Reply:
Just is just a debugging task, I found that I was always adding this to debug, so added it here.

Reply:
And what does `--cors *` do?

Reply:
So by default, it's set to block CORS calls, so if I am testing API locally from webhint.io (from a different localhost port), then the calls will get rejected. This will allow that so I can test locally or from webhint.io or postman